components/mpas-ocean/docs/user-guide/index.md

```diff
@@ -0,0 +1,210 @@
+
+#### Checking initial conditions
+
+Key information to check regarding the input data typically include:
+
+```text
+<immutable_stream name="mesh"
+                  type="none"
+                  io_type="pnetcdf"
+                  filename_template="/lcrc/group/e3sm/data/inputdata/ocn/mpas-o/IcoswISC30E3r5/mpaso.IcoswISC30E3r5.20231120.nc"
+/>
+<immutable_stream name="input"
+                  type="input"
+                  io_type="pnetcdf"
+                  input_interval="initial_only"
+                  filename_template="/lcrc/group/e3sm/data/inputdata/ocn/mpas-o/IcoswISC30E3r5/mpaso.IcoswISC30E3r5.20231120.nc"
+/>
+```
+
+The `mesh` filename points to the mesh file used. The `input` filename points to the file containing the ocean initial conditions (if the run type is `initial`).
+The streams file can be large, it is often useful to rely on the search function to navigate it. For larger meshes (millions of horizontal cells) the flag `io_type="pnetcdf"` must be changed to `io_type="pnetcdf,cdf5"`.
+
+#### Checking and modifying the output data
+
+By default, MPAS-Ocean will output a set of monthly-averaged variables. The streams file can be modified to include additional variables in the existing output files, produce additional output files, or change the output frequency (e.g. high frequency files shifting between daily or 5-daily frequencies).
+
+The XML file is organized into blocks describing each stream. Typical streams for output include:
+
+`timeSeriesStatsMonthlyOutput` - monthly averaged output
+
+`highFrequencyOutput` - high frequency snapshots (not averaged) output with frequency `output_interval`
+
+Under each block header is the list of variables (individual variables, variable structure, or variable arrays) that will be output within the relevant stream.
+
+##### Example workflow for modifying the output fields
+
+- copy the `streams.ocean` file from the run directory to the `SourceMods` directory (see above)
+- identify the variable name for the variable of interest. You can find the variable name by searching the ocean Registry.xml (in the `src` directory) or Registry_package.xml in the `tracer` and `analysis_member` sub-directories. Note whether the variable of interest is included within a `var_array` or a `var_struct`.
+- identify the output stream of interest (e.g. monthly averages, high frequency, others). You can search for a stream name, known output filename, or output interval.
+- check whether the variable of interest is included in the `ocean.streams` file. Search for the `var name`, or the `var_array` and `var_struct` if applicable. If it is, copy the variable line from other streams into the stream of interest. If it is not included, copy it from the Registry.xml.
```

Comment:
should we have `streams.ocean` here instead of `ocean.streams`

Reply:
done, thanks.